sfge/src/main/java/com/salesforce/rules/fls/apex/operations/FlsValidationCentral.java

```diff
@@ -156,18 +155,31 @@ public void createExpectedValidations(DmlStatementVertex vertex, SymbolProvider
         final BaseSFVertex childVertex = dmlStatementVertexChildren.get(0);
 
         final ValidationConverter validationConverter = new ValidationConverter(validationType);
-        final Set<FlsValidationRepresentation> validationReps;
+        final Set<FlsValidationRepresentation> validationReps = new HashSet<>();
         if (childVertex instanceof ChainedVertex) {
             final Optional<ApexValue<?>> apexValue =
                     ScopeUtil.resolveToApexValue(symbols, (ChainedVertex) childVertex);
             if (apexValue.isPresent()) {
-                validationReps = validationConverter.convertToExpectedValidations(apexValue.get());
+                validationReps.addAll(
+                        validationConverter.convertToExpectedValidations(apexValue.get()));
             } else {
-                throw new TodoException(
-                        "Apex value not detected for dml's child vertex: " + childVertex);
+                if (LOGGER.isWarnEnabled()) {
+                    LOGGER.warn(
+                            "TODO: Apex value not detected for dml's child vertex: " + childVertex);
+                    // TODO: add telemetry
+                }
+                violations.add(
+                        FlsViolationCreatorUtil.createUnresolvedCrudFlsViolation(
+                                validationType, vertex));
             }
         } else {
-            throw new TodoException("Child vertex of DML is not a chained vertex: " + childVertex);
+            if (LOGGER.isWarnEnabled()) {
+                LOGGER.warn("TODO: Child vertex of DML is not a chained vertex: " + childVertex);
+                // TODO: add telemetry
+            }
+            violations.add(
+                    FlsViolationCreatorUtil.createUnresolvedCrudFlsViolation(
+                            validationType, vertex));
```

Comment:
This is possibly an unexpected situation - we haven't seen it anywhere or had a repro, but I'm modifying it so that users don't see an internal error for this.